lib/membrane/srtp_keying_material_event.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Membrane.SRTP.KeyingMaterial.Event do
```

Comment:
I am wondering whether `Membrane.SRTP` is a good prefix because we are aliasing it by `alias Membrane.SRTP` and doing this in `membrane_rtp_plugin` suggests that this is something from `membrane_rtp_plugin` repo so it can be missleading.

Reply:
We do the same with `Membrane.RTP` and all other formats. Not sure if that's a real problem, but if so, we'd have to do something on the general level.